src/02-common-principles.md

```diff
@@ -446,10 +446,14 @@ exponent. TSV files MUST be in UTF-8 encoding.
 Example:
 
 ```Text
-onset duration  response_time correct stop_trial  go_trial
-200 200 0 n/a n/a n/a
+onset	duration	response_time	correct	stop_trial	go_trial
+200	200	0	n/a	n/a	n/a
 ```
 
+**Note**: TSV examples in the document can be re-formatted using
+spaces instead of tabs for human readability, and have tabs replaced
+with spaces by used for rendering HTML or PDF.
+
```

Comment:
I would rephrase this note:

```suggestion
**Note**: The TSV examples in this document (like the one above this note)
are occasionally formatted using space characters instead of tabs to improve
human readability.
Directly copying and then pasting these examples from the specification
for use in new BIDS datasets can lead to errors and is discouraged.
```

Reply:
thanks! accepted